docs/school-check-data.md

```diff
@@ -0,0 +1,65 @@
+# Generating School Check Data
+
+**NB: This process involves accessing the production console. You will need an
+approved PIM request, and a second developer to observe during this.**
+
+As part of verifying a teacher's claim service operators will send emails to the
+relevant schools to check their employment status. Generating a list of claims
```

Comment:
It's not strictly speaking employment status, it's a bit more complicated than that and varies per policy. Let's not be too specific here so as not to mislead:

```suggestion
relevant school to confirm their eligibility. Generating a list of claims
```